### PR TITLE
docs: serve mkdocs, align with chronicle-examples diagrams, and fix code blocks

### PR DESCRIPTION
**Pull request by suchapalaver**
[CHRON-218](https://blockchaintp.atlassian.net/browse/CHRON-218?atlOrigin=eyJpIjoiMTdjZjMwZjQ4Y2VhNGI5NjhkYTAzZmE0ODdmODZlZmMiLCJwIjoiaiJ9)

Signed-off-by: Joseph Livesey [joseph.livesey@btp.works](mailto:joseph.livesey@btp.works)

[CHRON-218]: https://blockchaintp.atlassian.net/browse/CHRON-218?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/recording_provenance.md [hunk 1]
```diff
@@ -194,44 +194,44 @@ the result of an operation should await the
...
- ```
+```
 
  And the second, when the Chronicle transaction has been committed to the
  ledger:
 
- ```json
+```json
 {
-  "stage": "COMMIT",
```
Comment (mtbc) [inline docs/recording_provenance.md:211]: I have my doubts about the re-spacing of this JSON.

Comment (suchapalaver) [reply docs/recording_provenance.md:211]: The reason for the editing was that the formatting completely off in the compiled docs. This PR at least fixes that issue, so now we have an actual code block. How would you change things now?

Comment (suchapalaver) [reply docs/recording_provenance.md:211]: Looking at it, I see the JSON formatter I used removed spaces in an unintended way. Anything else?

Comment (suchapalaver) [reply docs/recording_provenance.md:211]: Got it back to where we were!

**Review by mtbc — APPROVED** [review]

File: .github/workflows/cargo-checkmate.yaml [hunk 2]
```diff
@@ -24,4 +24,4 @@ jobs:
       - run: cargo-checkmate clippy
       - run: cargo-checkmate build
       - run: cargo-checkmate doc
-      - run: cargo-checkmate audit
+      - run: RUST_BACKTRACE=1 cargo-checkmate audit
```
Comment (mtbc) [inline .github/workflows/cargo-checkmate.yaml:27]: Do we still need this?

Comment (suchapalaver) [reply .github/workflows/cargo-checkmate.yaml]: Thanks, Mark!

**Review by mtbc — APPROVED** [review]

**Pull request merged**